fastlane/Fastfile

```diff
@@ -0,0 +1,33 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+
+default_platform(:android)
+
+platform :android do
+  desc "Build and deploy on Google Play"
+  lane :deploy do |options|
+    gradleTaskName = "assemble#{options[:flavor]}Release"
+    version = ENV['TRAVIS_TAG'].empty? ? 'SNAPSHOT' : ENV['TRAVIS_TAG']
+    package_name = options[:flavor] == 'unbranded' ? "org.medicmobile.webapp.mobile" : "org.medicmobile.webapp.mobile.#{options[:flavor]}"
+
+    gradle(task: gradleTaskName)
+
+    supply(
+      package_name: package_name,
+      track: "alpha",
```

Comment:
I wonder if we should go straight to beta at this point? I mean, it's only run when tagging so presumably it's reasonably stable. I think of alpha as more like nightly builds.

Not a big deal either way, we can define "alpha" however we like.

Reply:
From what I've seen, a lot of the apps use the "beta" channel as "production" and they have no active production versions. This is why I elected to go with alpha.